staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -153,13 +153,20 @@ func (s *store) Get(ctx context.Context, key string, opts storage.GetOptions, ou
 
 // Create implements storage.Interface.Create.
 func (s *store) Create(ctx context.Context, key string, obj, out runtime.Object, ttl uint64) error {
+	trace := utiltrace.New("Create",
+		utiltrace.Field{"key", key},
+		utiltrace.Field{"type", getTypeName(obj)},
+	)
+	defer trace.LogIfLong(500 * time.Millisecond)
 	if version, err := s.versioner.ObjectResourceVersion(obj); err == nil && version != 0 {
 		return errors.New("resourceVersion should not be set on objects to be created")
 	}
 	if err := s.versioner.PrepareObjectForStorage(obj); err != nil {
 		return fmt.Errorf("PrepareObjectForStorage failed: %v", err)
 	}
+	trace.Step("About to Encode")
```

Comment:
Since you add encoding - don't we want that in GuaranteedUpdate too?

Reply:
Done